src/modules/musig/main_impl.h

```diff
@@ -65,10 +65,10 @@ static void secp256k1_musig_coefficient(secp256k1_scalar *r, const unsigned char
      * equivalent to hashing the public key. Because the public key can be
      * identified by the index given the ordered list of public keys (included in
      * ell), the index is just a different encoding of the public key.*/
```

Comment:
This seems to be an argument in favor of hashing indices. What are the subtle issues with protocols building on threshold sigs?

Reply:
Hm, I don't know and I don't remember writing that paragraph. Perhaps @apoelstra knows.

Reply:
I believe I had some hypothetical variant of Russell's attack in mind here -- but since all participants are signing the same message, it doesn't apply. I was just confused.